src/Microsoft.Data.Analysis/PrimitiveDataFrameColumnArithmetic.tt

```diff
@@ -6,7 +6,6 @@
 <#@ import namespace="System.Collections.Generic" #>
```

Comment:
Is there still a benefit to keeping this TT file around?

Reply:
This file generates concrete implementations for required operations with supported types instead of generic. It should go away with using .Net 8 and generic math